ehitchhiking-ui/src/app/shared/components/trips-modal/trips-modal.component.sass

```diff
@@ -3,22 +3,8 @@
 @import '../../../../styles/mixins'
```

Comment:
'constants', 'fonts-constants', 'mixins' are already exist in trips. Pls remove extra imports

Reply:
fixed